packages/framework/src/Model/Product/Accessory/ProductAccessoryRepository.php

```diff
@@ -39,8 +39,12 @@ public function getProductAccessoryRepository()
      * @param int|null $limit
      * @return \Shopsys\FrameworkBundle\Model\Product\Product[]
      */
-    public function getTopOfferedAccessories(Product $product, $domainId, PricingGroup $pricingGroup, $limit)
-    {
+    public function getOfferedAccessories(
+        Product $product,
+        int $domainId,
+        PricingGroup $pricingGroup,
+        ?int $limit = ProductAccessoryFacade::PRODUCT_ACCESSORIES_FRONTEND_LIMIT,
```

Comment:
I recommend null as a default value here. 
And then limit only usages in `ProductEntityFieldMapper` and `ProductArrayFieldMapper`. 
The code will be shorter and the intention clearer imho.